src/main/java/blusunrize/immersiveengineering/client/render/IEOBJItemRenderer.java

```diff
@@ -124,38 +124,39 @@ else if(quadsForLayer.layer.isTranslucent())
 			Vector4f color = quadsForLayer.layer.getColor();
 			for(BakedQuad quad : quadsForLayer.quadsInLayer)
 				addQuadWithAlpha(
-						matrix.getLast(), quad, color, new int[]{light, light, light, light}, overlay, builder
+						matrix.getLast(), quad, color, light, overlay, builder
 				);
 			matrix.scale(1.01F, 1.01F, 1.01F);
 		}
 		matrix.pop();
 	}
 
 	private void addQuadWithAlpha(MatrixStack.Entry matrixEntryIn, BakedQuad quadIn, Vector4f color,
-								  int[] combinedLightsIn, int combinedOverlayIn, IVertexBuilder builder)
+								  int light, int combinedOverlayIn, IVertexBuilder builder)
 	{
-		int[] aint = quadIn.getVertexData();
+		int[] vertexData = quadIn.getVertexData();
 		Vec3i normalInt = quadIn.getFace().getDirectionVec();
 		Vector3f normal = new Vector3f(normalInt.getX(), normalInt.getY(), normalInt.getZ());
 		Matrix4f transform = matrixEntryIn.getMatrix();
 		normal.transform(matrixEntryIn.getNormal());
-		int i = 8;
-		int j = aint.length/8;
+
+		int intSize = DefaultVertexFormats.BLOCK.getIntegerSize();
+		int j = vertexData.length/intSize;
```

Comment:
As you already said on Discord, this is not an ideal name. I would suggest something like `numVertices`.

Reply:
That would work better, yes. I just named it somewhat after the method being called, which is also pretty poorly named really. I'll adjust that.